diff -Naru proj-9.6.2-orig/src/geodesic.c proj-9.6.2/src/geodesic.c
--- proj-9.6.2-orig/src/geodesic.c	2025-07-30 12:23:12.970344360 -0500
+++ proj-9.6.2/src/geodesic.c	2025-07-30 12:24:08.052165761 -0500
@@ -460,7 +460,7 @@
     (pS12 ? GEOD_AREA : GEOD_NONE);
 
   outmask &= l->caps & OUT_ALL;
-  if (!( (flags & GEOD_ARCMODE || (l->caps & (GEOD_DISTANCE_IN & OUT_ALL))) ))
+  if (!( (flags & GEOD_ARCMODE || ((unsigned)l->caps & ((unsigned)GEOD_DISTANCE_IN & (unsigned)OUT_ALL))) ))
     /* Impossible distance calculation requested */
     return NaN;
 
@@ -1072,7 +1072,7 @@
     azi1 = atan2dx(salp1, calp1);
   caps = caps ? caps : GEOD_DISTANCE_IN | GEOD_LONGITUDE;
   /* Ensure that a12 can be converted to a distance */
-  if (caps & (OUT_ALL & GEOD_DISTANCE_IN)) caps |= GEOD_DISTANCE;
+  if (caps & ((unsigned)OUT_ALL & (unsigned)GEOD_DISTANCE_IN)) caps |= GEOD_DISTANCE;
   geod_lineinit_int(l, g, lat1, lon1, azi1, salp1, calp1, caps);
   geod_setarc(l, a12);
 }
