diff -Naur proj-9.3.0.orig/test/cli/testvarious proj-9.3.0/test/cli/testvarious
--- proj-9.3.0.orig/test/cli/testvarious	2023-09-01 00:17:37.000000000 -0500
+++ proj-9.3.0/test/cli/testvarious	2023-10-17 14:11:29.068420888 -0500
@@ -1320,6 +1320,15 @@
 299905.060 4499796.515
 EOF
 
+echo "##############################################################" >> ${OUT}
+echo "Test PROJ_NETWORK=ON -d 3 EPSG:26915 +to EPSG:26715" >> ${OUT}
+PROJ_NETWORK=ON $EXE -d 3 EPSG:26915 +to EPSG:26715 -E >>${OUT} 2>&1 <<EOF
+569704.57 4269024.67
+EOF
+
+PROJ_NETWORK=ON $EXE -d 3 EPSG:26915 +to EPSG:26715 -E >>${OUT} 2>&1 <<EOF
+569704.57 4269024.67
+EOF
 
 # Done!
 # do 'diff' with distribution results
diff -Naur proj-9.3.0.orig/test/cli/tv_out.dist proj-9.3.0/test/cli/tv_out.dist
--- proj-9.3.0.orig/test/cli/tv_out.dist	2023-09-01 00:17:37.000000000 -0500
+++ proj-9.3.0/test/cli/tv_out.dist	2023-10-17 14:10:39.000283090 -0500
@@ -633,3 +633,7 @@
 ##############################################################
 Test inverse of Similarity Transformation (example from EPSG Guidance Note 7.2)
 299905.060 4499796.515	300000.000	4500000.000 0.000
+##############################################################
+Test PROJ_NETWORK=ON -d 3 EPSG:26915 +to EPSG:26715
+569704.57 4269024.67   569722.398  4268814.269 0.000
+569704.57 4269024.67   569722.398  4268814.269 0.000
